database/USAFacts_collector

```diff
@@ -0,0 +1,105 @@
+############################################################################################################################
+################################## Pulling and Transforming USAFacts County Level Data ##################################### 
+############################################################################################################################
+# Script for taking USAFacts county-level dataset, adding lat/long coordinates for each county, and transforming datasheet
+# into NECSI time-series format
+# Only one required package:
+library(readr)
+
+#### Set working directory ####
+setwd("~/Documents/Data_and_Projects/NECSI-COVID19/Data/USA_CountyLevel")
+
+#### Load data ####
+### USAFacts
+cases_url <- "https://static.usafacts.org/public/data/covid-19/covid_confirmed_usafacts.csv?_ga=2.104843072.1960452769.1585151006-660813358.1585151006"
+covid_usa_cases <- read_csv(url(cases_url))
+
+deaths_url <- "https://static.usafacts.org/public/data/covid-19/covid_deaths_usafacts.csv?_ga=2.62244076.1960452769.1585151006-660813358.1585151006"
+covid_usa_deaths <- read_csv(url(deaths_url))
+
+### Dataset with lat/long county coordinates 
+county_latlong_url <- "https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_daily_reports/03-24-2020.csv"
```

Comment:
This should not be hardcoded to a specific day

Reply:
It's only using the lat/long coordinates in that data sheet. Olga had forwarded it to me as a place to get county-level lat/long coordinates.

Reply:
Oh, I see. Can you add a comment then?

Reply:
Done!